minissdpd/minissdpd.c

```diff
@@ -472,7 +472,7 @@ SendSSDPMSEARCHResponse(int s, const struct sockaddr * sockname,
 	 *
 	 * have a look at the document "UPnP Device Architecture v1.1 */
 	l = snprintf(buf, sizeof(buf), "HTTP/1.1 200 OK\r\n"
-		"CACHE-CONTROL: max-age=120\r\n"
+		"CACHE-CONTROL: max-age=1800\r\n"
```

Comment:
why changing from 2 to 30 minutes ?

Reply:
https://openconnectivity.org/upnp-specs/UPnP-arch-DeviceArchitecture-v2.0-20200417.pdf
p39 1.3.3 Search response

> CACHE-CONTROL
> Required. Field value shall have the max-age directive (“max-age=”) followed by an integer that specifies the
> number of seconds the advertisement is valid. After this duration, control points should assume the device (or
> service) is no longer available; as long as a control point has received at least one advertisement that is still
> valid from a root device, any of its embedded devices or any of its services, then the control point can assume
> that all are available. The number of seconds should be greater than or equal to 1800 seconds (30 minutes),
> although exceptions are defined in the text above. Specified by UPnP vendor. Other directives shall notbe sent
> and shall be ignored when received